Gulpfile.js

```diff
@@ -107,6 +112,7 @@ const projects = [
         cwd: "packages/table/",
         dependencies: ["core"],
         copy: false,
+        isotest: true,
```

Comment:
this change may not be worth it--it's just easier and more flexible to have separate flags.